cnf-certification-test/certification/certtool/certtool.go

```diff
@@ -30,7 +30,7 @@ const (
 )
 
 var (
-	CertAPIClient api.CertificationValidator
+	CertAPIClient certdb.ResourceCertificator
```

Comment:
I don't think this variable is needed anymore. Same with the functions that use it in this file: they seem not to be used:
GetContainerCertificationRequestFunction
GetOperatorCertificationRequestFunction
WaitForCertificationRequestToSuccess